app/assets/stylesheets/application/member-card.scss

```diff
@@ -1,16 +1,34 @@
+/* Uso de Flexbox para organizar el contenedor de los cards*/
+.row.to-member-cards {
```

Comment:
nesting in css usually is not a good practice because we want to write a css that it is as reusable as possible and if we use that we won't be able to use the class without the other one